deep-learning/_posts/2018-10-12-Sentinel2SuperResolution.md

```diff
@@ -0,0 +1,60 @@
+---
+ layout: review
+ title:  " Super-resolution of Sentinel-2 images: Learning a globally applicable deep neural network"
+ tags:   deep-learning, remote-sensing, super-resolution, hyperspectral
+ author: Charles Authier
+ pdf:  https://arxiv.org/abs/1803.04271v2
+ cite:
+   authors: "Charis Lanaras, José Bioucas-Dias, Silvano Galliani, Emmanuel Baltsavias, Konrad Schindler"
+   title:   "Super-resolution of Sentinel-2 images: Learning a globally applicable deep neural network"
+   venue:   "Arxiv, 12 Mar 2018 and ISPRS Journal of Photogrammetry and Remote Sensing, 146 (2018), pp. 305-319"
+ ---
+
+A big problem in deep-learning is the lak of data, in remote-sensing not only the data available is very small, they don't necesery have the same resolution. 
+
+
+A lot of peper show a real use of the hyperspectral in deep-learning, unfortunally, txecpt for small hyperspectral use for land or aerial imaging, the resolution for those hyperspectral sensors are generally wrose than a RGB sensor.
+
+
+This paper show a super-resolution method to put the hyperspectral band with a larger GSD to the same 10m GSD than the RGB. Those images can be use for better analyse in earth science or in our case for adding more information on the images for better prediction. 
```

Comment:
shows 
to the same resolution than that of the RGB image.
be used to better analyse data and thus improve prediction accuracy.